README.md

```diff
@@ -8,15 +8,24 @@ Installing by compilation is very straight forward. The following dependencies a
 - Go
 - GCC
 
-### Method 1 - Download and install
+### Method 1 - Fresh Install
+If you need to install vroomy:
 ```bash
-curl -s https://raw.githubusercontent.com/vroomy/vroomy/master/install | bash -s
```

Comment:
Why can't we just update the script?

Reply:
bin/install is now a compile -X(flags) command that can't run from inside go

we can add a new script if you like?

Reply:
added a new script :)